pandas/core/groupby.py

```diff
@@ -3468,6 +3459,24 @@ def _apply_to_column_groupbys(self, func):
              in self._iterate_column_groupbys()),
             keys=self._selected_obj.columns, axis=1)
 
+    def count(self):
```

Comment:
Having an axis argument there is a bug when [current master simply ignores the axis arg; `groupby.py:L807`](https://github.com/pydata/pandas/blob/81b647f4e623446d7b584e216382e79c2ed2e720/pandas/core/groupby.py#L807)

also, when doing a groupby, counting on `axis=1` has no meaning because u may get a different values for each row regardless of keys being the same or different.